app/views/registrations_mailer/notify_delegates_of_formerly_banned_user_registration.erb

```diff
@@ -6,7 +6,15 @@
   just registered for <%= @registration.competition.name %>.
 </p>
 
-<p>
-You can check the ban scope and reason
-  <%= link_to "here", panel_page_url(id: User.panel_pages[:bannedCompetitors]) %>
-</p>
+<% user = @registration.user %>
+<% bans = user.past_roles.filter { |role| role.group == UserGroup.banned_competitors.first } %>
```

Comment:
May I jump in and try to find a cleaner association link for this?

Reply:
Absolutely!

Reply:
I just used what we had in the `user#banned_in_past?` method

Reply:
Which is admittedly also not very optimized at all. But that's for another PR!

Reply:
Please try and see whether the association that I just pushed works.
(Also do we have tests for this?)

Reply:
Oooh I like defining it as an association

Reply:
We don't have tests against this at the moment - want me to add some?

Reply:
Also, the strings could be i18n'd - but perhaps we can leave that and tests to volunteer contributors? And call this a hotfix?

Reply:
oKIDOKI!